tests/buildkite/benchmark/test_run_benchmarks.py

```diff
@@ -213,4 +214,98 @@ def test_run_benchmarks():
         run = MockRun(repo, test["run_filters"])
         run.benchmarkable_type = "arrow-commit"
         run.run_all_benchmark_groups()
-        assert run.executed_commands == test["expected_commands"]
+        assert run.executor.executed_commands == test["expected_commands"]
+
+
+def test_run_arrowbench_benchmarks():
+    repo = deepcopy(repos_with_benchmark_groups[1])
```

Comment:
Could you pick the one with the correct repo name instead of the second one? Just so it's easier to debug if we change this big list later

Reply:
Yeah, this is how the previous test was, but it's brittle; I'll fix it

Reply:
Ok this went a bit weird and I can't fully explain why, but I'm switching to `not all(bg.mock run ...)` because it works.